CoffeeChat/Controller/TabPageViewController.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  TabPageViewController.swift
+//  CoffeeChat
+//
+//  Created by Lucy Xu on 10/1/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class TabPageViewController: UIPageViewController {
+
+    // MARK: - Private View Vars
+    private var matchViewController: MatchViewController!
```

Comment:
We could avoid force unwrapping these by initializing them in the init, so if we forget to set one it'll be caught at build time

Although we do a similar pattern in other places in the project, so its up to you if you think its worth changing